tests/test_standard_tags.py

```diff
@@ -38,7 +38,6 @@ def test_method_trace(self):
 @released(dotnet="2.13.0", golang="1.40.0", java="0.107.1", nodejs="3.0.0")
 @released(php="0.76.0", python="1.6.0rc1.dev", ruby="?")
 @rfc("https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2490990623/QueryString+-+Sensitive+Data+Obfuscation")
-@bug(weblog_variant="spring-boot-undertow", reason="APMJAVA-877")
```

Comment:
MMh, now it's removed? I may have added a dummy comment, or is it some merge issue ?